src/Endpoints/ZoneSettings.php

```diff
@@ -106,6 +106,37 @@ public function getHotlinkProtectionSetting($zoneID)
         return false;
     }
 
+    public function getBrowserCacheTtlSetting($zoneID)
+    {
+        $return = $this->adapter->get(
+            'zones/' . $zoneID . '/settings/browser_cache_ttl'
+        );
+        $body   = json_decode($return->getBody());
+
+        if ($body->success) {
+            return $body->result->value;
+        }
+
+        return false;
+    }
+
+    public function updateBrowserCacheTtlSetting($zoneID, $value)
+    {
+        $return = $this->adapter->patch(
+            'zones/' . $zoneID . '/settings/browser_cache_ttl',
+            [
+                'value' => $value
+            ]
+        );
+        $body   = json_decode($return->getBody());
+
+        if ($body->success) {
+            return true;
+        }
+
+        return false;
```

Comment:
You could probably `return $body->success` here and save the conditional above. 😄

Reply:
@anthonysterling thanks for your time.
I thought better to be a consistent approach and I did same functions way in the repo :)